Src/Support/GoogleApis/Apis/[Media]/Upload/ResumableUpload.cs

```diff
@@ -522,15 +643,16 @@ protected virtual void ProcessResponse(HttpResponseMessage httpResponse)
                 }.CreateRequest();
 
             // Prepare next chunk to send.
-            if (StreamLength != UnknownSize)
+            int contentLength;

```

Comment:
Hmm, maybe I've backed myself into a corner - ?: *is* an 'if' expression. OK, I'll change it...